cmd/sonobuoy/app/status.go

```diff
@@ -127,30 +164,38 @@ func printAll(w io.Writer, status *aggregation.Status) error {
 }
 
 func printSummary(w io.Writer, status *aggregation.Status) error {
-	tw := tabwriter.NewWriter(w, 1, 8, 1, '\t', tabwriter.AlignRight)
+	tw := defaultTabWriter(w)
 	totals := map[string]map[string]int{}
-	for _, plugin := range status.Plugins {
-		if _, ok := totals[plugin.Plugin]; !ok {
-			totals[plugin.Plugin] = make(map[string]int)
+
+	// Effectively making a pivot chart to count the unique combinations of status/result.
+	statusResultKey := func(p aggregation.PluginStatus) string {
+		return p.Status + ":" + p.ResultStatus
```

Comment:
We were just iterating over values and rolling on on some value before (status). Now we roll up on the tuple (status, resultStatus).